src/debug.c

```diff
@@ -436,6 +436,8 @@ void debugCommand(client *c) {
             "CLOSE-CLUSTER-LINK-ON-PACKET-DROP <0|1>",
             "    This is valid only when DROP-CLUSTER-PACKET-FILTER is set to a valid packet type.",
             "    When set to 1, the cluster link is closed after dropping a packet based on the filter.",
+            "CLUSTER-RANDOM-MIN-PING <0|1>",
+            "    Enable or disable the cluster random min ping.",
```

Comment:
Enable is 0 and disable is 1? Or reverse?

Reply:
1 is enable and 0 is disable. My wording is a bit bad.

Reply:
I suggest a more little bit more detailed explanation. It also helps to say that it is enabled by default. All the other ones are off by default.

```suggestion
            "    Send cluster ping to a random node every second. Enabled by default.",
```